src/main/java/com/partymusicapp/service/PartyService.java

```diff
@@ -1,7 +1,20 @@
 package com.partymusicapp.service;
 
+import com.partymusicapp.models.dto.PartyDTO;
 import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
 
 @Service
 public interface PartyService {
+
+    PartyDTO getParty(String partyId);
+
+    void updateParty(@PathVariable String partyId);
```

Comment:
Nu mai trebuie adnotari cu @PathVariable si @RequestBody in service. Numa la controller ca sa stie de unde le ia, din URL ADRESS sau din BODY